modules/utxo-lib/test/bitgo/signature.ts

```diff
@@ -80,3 +81,22 @@ describe('signature', function () {
       scriptTypes2Of3.forEach((scriptType) => runTest(network, scriptType));
     });
 });
+
+describe('Signature (p2shP2pkh)', function () {
+  it('sign and parse', function () {
+    const signedTransaction = getTransactionBuilder(
+      fixtureKeys,
+      fixtureKeys.slice(0, 1),
+      'p2shP2pkh',
+      networks.bitcoin
+    ).build();
+
+    signedTransaction.ins.forEach((input) => {
+      assert.deepStrictEqual(parseSignatureScript(input), {
+        inputClassification: 'scripthash',
+        isSegwitInput: false,
+        p2shOutputClassification: 'pubkey',
```

Comment:
I'm also guessing this should be `pubkeyhash`, unless in fact the RP inputs are p2sh-p2pk and not p2sh-p2pkh.

Reply:
actually the RP inputs ARE `p2sh-p2pk`